src/functionalTest/kotlin/com/cognifide/gradle/aem/test/AemBuildResult.kt

```diff
@@ -17,10 +17,16 @@ class AemBuildResult(val result: BuildResult, val projectDir: File) {
 
     fun assertFileExists(path: String) = assertFileExists(file(path))
 
+    fun assertFileDoesntExist(path: String) = assertFileDoesntExist(file(path))
+
     fun assertFileExists(file: File) {
         assertTrue({ file.exists() }, "File does not exist: $file")
     }
 
+    fun assertFileDoesntExist(file: File) {
+        assertFalse({ file.exists() }, "File exist but should be removed: $file")
```

Comment:
`should be removed`? ... in some cases this assertion could lie, this could be also precondition without any removal before

Reply:
OK